src/coreclr/vm/peimagelayout.cpp

```diff
@@ -223,6 +223,7 @@ void PEImageLayout::ApplyBaseRelocations(bool relocationMustWriteCopy)
 
         BYTE * pageAddress = (BYTE *)GetBase() + rva;
 
+#if !defined(__APPLE__) && !defined(HOST_ARM64)
```

Comment:
I think this condition is incorrect and it should be:
```suggestion
#if !(defined(__APPLE__) && defined(HOST_ARM64))
```

Reply:
agreed, thanks!